bin/verify-deployment

```diff
@@ -21,8 +21,8 @@ UPDATED_REPLIACS=-1
 REPLICAS=0
 until [ "$UPDATED_REPLIACS" == "$REPLICAS" ]; do
   sleep 1
-  REPLICAS=$(kubectl get deployment ${1} --namespace=${2:=default} -o yaml | grep replicas | sed -n 2p | awk '{print $2}')
-  UPDATED_REPLIACS=$(kubectl get deployment ${1} --namespace=${2:=default} -o yaml | grep updatedReplicas | awk '{print $2}')
+  REPLICAS=$(kubectl get deployment ${1} --namespace=${2:=default} -o yaml | grep replicas: | sed -n 2p | awk '{print $2}')
+  UPDATED_REPLIACS=$(kubectl get deployment ${1} --namespace=${2:=default} -o yaml | grep updatedReplicas: | awk '{print $2}')
```

Comment:
@philipcristiano can you give me your thoughts on this change? this was added when I had a deployment with `replicas` in the name and the grep got stuck on the wrong value.